pymc/logprob/transforms.py

```diff
@@ -672,7 +672,10 @@ def find_measurable_transforms(fgraph: FunctionGraph, node: Node) -> Optional[Li
 
     # Do not apply rewrite to discrete variables
```

Comment:
```suggestion
    # Do not apply rewrite to discrete variables other than add and negation
```

Reply:
We need to explicitly raise NotImplementedError for cdf/icdf of measurable discrete variables as the existing logic for those is only correct for continuous ones: https://github.com/pymc-devs/pymc/issues/6984